src/main/java/org/isf/shared/GenericMapper.java

```diff
@@ -89,7 +89,8 @@ public PageInfoDTO setParameterPageInfo(PageInfo pageInfo) {
 		pageInfoDTO.setHasNextPage(pageInfo.isHasNextPage());
 		pageInfoDTO.setHasPreviousPage(pageInfo.isHasPreviousPage());
 		pageInfoDTO.setSize(pageInfo.getSize());
-		pageInfoDTO.setTotalCount(pageInfo.getTotalCount());
+		pageInfoDTO.setTotalNbOfElements(pageInfo.getTotalNbOfElements());
+		pageInfoDTO.setTotalPage(pageInfo.getTotalPage());
```

Comment:
this seems to be the right method name, so far...

```suggestion
		pageInfoDTO.setTotalPage(pageInfo.getTotalPages());
```